tests/src/test/scala/cats/tests/FunctionTests.scala

```diff
@@ -25,4 +26,6 @@ class FunctionTests extends CatsSuite {
 
   checkAll("Function1[Int, Int]", ContravariantTests[? => Int].contravariant[Int, Int, Int])
   checkAll("Contravariant[? => Int]", SerializableTests.serializable(Contravariant[? => Int]))
+
+  checkAll("Function1[String, Int]", GroupLaws[Function1[String, Int]].monoid)
```

Comment:
You've probably checked for this, but just to make sure :-) My first inclination is there's no `SerializableTests` because it can't be serialized, though it seems we're testing for it in the `Contravariant` instance?